packages/graz/src/actions/methods.ts

```diff
@@ -104,6 +104,24 @@ export const sendIbcTokens = async ({
   );
 };
 
+// https://cosmos.github.io/cosmjs/latest/stargate/classes/SigningStargateClient.html#signAndBroadcast
+export interface SignAndBroadcastArgs {
+  signerAddress?: string;
+  messages: readonly EncodeObject[];
+  fee: number | StdFee | "auto";
+  memo?: string;
+}
+
+export const SignAndBroadcast = async ({ signerAddress, messages, fee, memo }: SignAndBroadcastArgs) => {
```

Comment:
this is a function, please use lowercase on the first letter `signAndBroadcast`

Reply:
I don't think we don't need this function, we can execute it inside the mutation

Reply:
just want to confirm, what you mean is we don't need `signAndBroadcast ` function?

Reply:
yes, please check `sendTokens` as reference